src/components/NcModal/NcModal.vue

```diff
@@ -714,10 +722,13 @@ export default {
 				allowOutsideClick: true,
 				fallbackFocus: contentContainer,
 				trapStack: getTrapStack(),
+				// Esc can be used without stop in content or additionalTrapElements where it should not deacxtivate modal's focus trap.
+				// Focus trap is deactivated on modal close anyway.
+				escapeDeactivates: false,
```

Comment:
```suggestion
```
Not used anymore?

Reply:
Hmm... I'm not sure if it is required, I need to re-check.

But disabling `escapeDeactivates` is definitely safe:
- If a modal is closed by ESC, trap is deactivated anyway
- If a modal is not closed by ESC, the focus trap should also not be deactivated.